[2025-01-21T18:35:50.156+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-01-21T18:35:50.171+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: DAG_AWS_PIPELINE.extract_events manual__2025-01-21T18:35:45.730388+00:00 [queued]>
[2025-01-21T18:35:50.179+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: DAG_AWS_PIPELINE.extract_events manual__2025-01-21T18:35:45.730388+00:00 [queued]>
[2025-01-21T18:35:50.180+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 2
[2025-01-21T18:35:50.193+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): extract_events> on 2025-01-21 18:35:45.730388+00:00
[2025-01-21T18:35:50.203+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=3712) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-01-21T18:35:50.204+0000] {standard_task_runner.py:72} INFO - Started process 3722 to run task
[2025-01-21T18:35:50.205+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'DAG_AWS_PIPELINE', 'extract_events', 'manual__2025-01-21T18:35:45.730388+00:00', '--job-id', '361', '--raw', '--subdir', 'DAGS_FOLDER/DAG_AWS_PIPELINE.py', '--cfg-path', '/tmp/tmpy7clww7z']
[2025-01-21T18:35:50.206+0000] {standard_task_runner.py:105} INFO - Job 361: Subtask extract_events
[2025-01-21T18:35:50.251+0000] {task_command.py:467} INFO - Running <TaskInstance: DAG_AWS_PIPELINE.extract_events manual__2025-01-21T18:35:45.730388+00:00 [running]> on host 59ff16f1d4fb
[2025-01-21T18:35:50.334+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Wilfrido Rondon' AIRFLOW_CTX_DAG_ID='DAG_AWS_PIPELINE' AIRFLOW_CTX_TASK_ID='extract_events' AIRFLOW_CTX_EXECUTION_DATE='2025-01-21T18:35:45.730388+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-01-21T18:35:45.730388+00:00'
[2025-01-21T18:35:50.335+0000] {logging_mixin.py:190} INFO - Task instance is in running state
[2025-01-21T18:35:50.335+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2025-01-21T18:35:50.335+0000] {logging_mixin.py:190} INFO - Current task name:extract_events state:running start_date:2025-01-21 18:35:50.171538+00:00
[2025-01-21T18:35:50.336+0000] {logging_mixin.py:190} INFO - Dag name:DAG_AWS_PIPELINE and current dag run status:running
[2025-01-21T18:35:50.336+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-01-21T18:35:50.336+0000] {DAG_AWS_PIPELINE.py:119} INFO - Recuperando página 1...
[2025-01-21T18:35:50.749+0000] {DAG_AWS_PIPELINE.py:129} INFO - Reportes recuperados: 10, total acumulado: 10
[2025-01-21T18:35:50.749+0000] {DAG_AWS_PIPELINE.py:119} INFO - Recuperando página 2...
[2025-01-21T18:35:51.235+0000] {DAG_AWS_PIPELINE.py:129} INFO - Reportes recuperados: 10, total acumulado: 20
[2025-01-21T18:35:51.235+0000] {DAG_AWS_PIPELINE.py:119} INFO - Recuperando página 3...
[2025-01-21T18:35:51.556+0000] {DAG_AWS_PIPELINE.py:134} WARNING - Rate limit alcanzado. Esperando 60 segundos...
[2025-01-21T18:36:48.330+0000] {DAG_AWS_PIPELINE.py:119} INFO - Recuperando página 3...
[2025-01-21T18:36:48.786+0000] {DAG_AWS_PIPELINE.py:129} INFO - Reportes recuperados: 10, total acumulado: 30
[2025-01-21T18:36:48.786+0000] {DAG_AWS_PIPELINE.py:119} INFO - Recuperando página 4...
[2025-01-21T18:36:49.421+0000] {DAG_AWS_PIPELINE.py:129} INFO - Reportes recuperados: 10, total acumulado: 40
[2025-01-21T18:36:49.421+0000] {DAG_AWS_PIPELINE.py:119} INFO - Recuperando página 5...
[2025-01-21T18:36:49.739+0000] {DAG_AWS_PIPELINE.py:134} WARNING - Rate limit alcanzado. Esperando 60 segundos...
[2025-01-21T18:37:46.413+0000] {DAG_AWS_PIPELINE.py:119} INFO - Recuperando página 5...
[2025-01-21T18:37:46.787+0000] {DAG_AWS_PIPELINE.py:129} INFO - Reportes recuperados: 10, total acumulado: 50
[2025-01-21T18:37:46.787+0000] {DAG_AWS_PIPELINE.py:119} INFO - Recuperando página 6...
[2025-01-21T18:37:47.151+0000] {DAG_AWS_PIPELINE.py:129} INFO - Reportes recuperados: 10, total acumulado: 60
[2025-01-21T18:37:47.151+0000] {DAG_AWS_PIPELINE.py:119} INFO - Recuperando página 7...
[2025-01-21T18:37:47.495+0000] {DAG_AWS_PIPELINE.py:129} INFO - Reportes recuperados: 10, total acumulado: 70
[2025-01-21T18:37:47.495+0000] {DAG_AWS_PIPELINE.py:119} INFO - Recuperando página 8...
[2025-01-21T18:37:47.903+0000] {DAG_AWS_PIPELINE.py:129} INFO - Reportes recuperados: 10, total acumulado: 80
[2025-01-21T18:37:47.904+0000] {DAG_AWS_PIPELINE.py:119} INFO - Recuperando página 9...
[2025-01-21T18:37:48.134+0000] {DAG_AWS_PIPELINE.py:129} INFO - Reportes recuperados: 10, total acumulado: 90
[2025-01-21T18:37:48.134+0000] {DAG_AWS_PIPELINE.py:119} INFO - Recuperando página 10...
[2025-01-21T18:37:48.358+0000] {DAG_AWS_PIPELINE.py:129} INFO - Reportes recuperados: 10, total acumulado: 100
[2025-01-21T18:37:48.358+0000] {DAG_AWS_PIPELINE.py:141} INFO - Transformando los datos a CSV...
[2025-01-21T18:37:48.366+0000] {DAG_AWS_PIPELINE.py:148} INFO - CSV guardado temporalmente en: /tmp/reports.csv
[2025-01-21T18:37:48.367+0000] {python.py:240} INFO - Done. Returned value was: /tmp/reports.csv
[2025-01-21T18:37:48.390+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-01-21T18:37:48.390+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=DAG_AWS_PIPELINE, task_id=extract_events, run_id=manual__2025-01-21T18:35:45.730388+00:00, execution_date=20250121T183545, start_date=20250121T183550, end_date=20250121T183748
[2025-01-21T18:37:48.399+0000] {logging_mixin.py:190} INFO - Task instance in success state
[2025-01-21T18:37:48.399+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: running
[2025-01-21T18:37:48.400+0000] {logging_mixin.py:190} INFO - Dag name:DAG_AWS_PIPELINE queued_at:2025-01-21 18:35:45.767546+00:00
[2025-01-21T18:37:48.400+0000] {logging_mixin.py:190} INFO - Task hostname:59ff16f1d4fb operator:PythonOperator
[2025-01-21T18:37:48.431+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-01-21T18:37:48.457+0000] {taskinstance.py:3895} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-01-21T18:37:48.460+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
